docs/content/operator/common/archives.mdx

```diff
@@ -1,21 +1,34 @@
----
-title: IOTA Archives
-description: The archive is a historical record of all transactions on IOTA. Enable archiving on your full nodes as a best practice.
-tags: [node-operation]
-teams:
-  - iotaledger/node
----
-
 import Quiz from '@site/src/components/Quiz';
 import questions from '/json/node-operators/archives.json';
 
+
+An IOTA archive instance stores the full IOTA transaction history since genesis in a database agnostic format. This includes information about transactions (with client authentication), effects, events, and checkpoints. As such, archival storage can be used for data auditing and for replaying historic transactions.
+
+:::info
+
+The current archival storage format doesn't include historic object versions. 
+
+:::
+
+As a full node operator, you can enable archival fallback for your full node by specifying the URL to upload archival data. To ensure a healthy network, we encourage the IOTA community to set up additional archives to ensure archival data availability across the network. In a typical configuration, an archive trails behind the latest checkpoint by approximately 10 minutes.
+
+A Full Node that starts from scratch can replay, and re-verify, transactions that occurred since IOTA genesis from the given archive via configuring Archival Fallback in the `fullnode.yaml` configuration file to point to the S3 bucket that stores the archive.
+
+An IOTA full node that fails to retrieve checkpoints from its peers via state sync protocol falls back to downloading the missing checkpoints from its pre-configured archive. This fallback enables a full node to catch up with the rest of the system regardless of the pruning policies of its peers.
+
+## IOTA Full Node Key-value Store Backup
+
+To enable historic data queries for the IOTA full nodes that prune old transactional data, full node RPC implementation is configured to fallback for querying missing transactional data from a remote store.
+
+If the information about the transaction digest, effects, events, or checkpoints is not available locally, a full node automatically retrieves the historical data from a cloud-based key-value store (currently managed by the IOTA Foundation). Note that the current key-value store implementation keeps historic transactional data only. We plan to provide support for a similar setup for retrieving the historic object versions in a future release.
```

Comment:
> We plan to provide support for a similar setup for retrieving the historic object versions in a future release.

Oh, do we plan that?

Reply:
Removed for now: https://github.com/iotaledger/iota/pull/5161/commits/c8c50399e79f8160b1f5b24aa4a8077251229ab9